libs/@local/hash-isomorphic-utils/src/flows/step-definitions.ts

```diff
@@ -0,0 +1,104 @@
+import type { ActionDefinition, TriggerDefinition } from "./types";
+
+export const triggerDefinitions = {
+  userTrigger: {
+    kind: "trigger",
+    name: "User Trigger",
+  },
+  userVisitedWebPageTrigger: {
+    kind: "trigger",
+    name: "User Visited Web Page Trigger",
+    outputs: [
+      {
+        payloadKind: "WebPage",
+        name: "visitedWebPage" as const,
+        array: false,
+      },
+    ],
+  },
+} satisfies Record<string, TriggerDefinition>;
+
+export const actionDefinitions = {
+  generateWebQuery: {
+    name: "Generate Web Query",
+    kind: "action",
+    inputs: [
+      {
+        oneOfPayloadKinds: ["Text"],
+        name: "prompt" as const,
```

Comment:
does `as const satisfies` on the whole object avoid having to add them individually?